httpx/_models.py

```diff
@@ -441,7 +441,7 @@ def __init__(self, *args: QueryParamTypes, **kwargs: typing.Any) -> None:
             items = parse_qsl(value)
         elif isinstance(value, QueryParams):
             items = value.multi_items()
-        elif isinstance(value, list):
+        elif type(value) in [list, tuple]:
```

Comment:
From the [docs for `type()`](https://docs.python.org/3.8/library/functions.html#type):
> The `isinstance()` built-in function is recommended for testing the type of an object, because it takes subclasses into account.

This would be better written as:
```suggestion
        elif any(isinstance(value, t) for t in [list, tuple]):
```

Reply:
Even better is this... :)

```python
if isinstance(... (tuple, list)): ...
```